tests/fixtures/cache_utils.py

```diff
@@ -0,0 +1,254 @@
+import os
+import shutil
+import json
+
+from models.base import FileOnDiskMixin
+from util.logger import SCLogger
+
+# ======================================================================
+# Functions for copying FileOnDisk objects to/from cache
+
+def copy_to_cache(FoD, cache_dir, filepath=None):
+    """Save a copy of the object (and associated files) into a cache directory.
+
+    If the object is a FileOnDiskMixin, then the file(s) pointed by get_fullpath()
+    will be copied to the cache directory with their original names,
+    unless filepath is specified, in which case the cached files will
+    have a different name than the files in the data folder (and the database filepath).
+    The filepath (with optional first extension) will be used to create a JSON file
+    which holds the object's column attributes (i.e., only those that are
+    database persistent).
+
+    If caching a non-FileOnDiskMixin object, the filepath argument must be given,
+    because it is used to name the JSON file.
+
+    Parameters
+    ----------
+    FoD: FileOnDiskMixin or None
+        The object (which is an instance of FileOnDiskMixin) to cache.
```

Comment:
If I'm caching a non-FileOnDiskMixin then you still have to give it the object to turn it into a JSON...